tests/Unit/Evolution/DgSubcell/Test_Mesh.cpp

```diff
@@ -13,6 +13,38 @@ void test_mesh() {
   constexpr size_t min_num_pts =
       Spectral::minimum_number_of_points<BasisType, QuadratureType>;
   constexpr size_t max_num_pts = Spectral::maximum_number_of_points<BasisType>;
+#ifdef SPECTRE_DEBUG
```

Comment:
check if the includes were updated here...  if Basis and Quadrature are not included, they should be now